blog/2024/04-28-jellyfin-release-10.9.0/index.mdx

```diff
@@ -0,0 +1,136 @@
+
+## The Big Changes
+
+This release has been over two years in the making, so we're really glad to finally get it out to you. The long cycle does mean quite an extensive changelog however, with well over 1100 pull requests merged into our `master` branch since 10.8.0 first dropped back in 2022. This section will detail all the big changes, broken up by general function and area of the system. In addition to these big items, there are hundreds of smaller bugfixes, quality-of-life tweaks, and other minor changes that we won't touch on here, but if you want to see the full list, you can do so over on the GitHub releases for [the core server](https://github.com/jellyfin/jellyfin/releases/tag/v10.9.0) and [the Web UI](https://github.com/jellyfin/jellyfin-web/releases/tag/v10.9.0).
+
+### General
+
+* We now support "trickplay" a.k.a. live video scrubbing. When scrubbing through a video with this enabled, you will be able to see a live preview of the video at that timestamp. Note that this requires explicit client support, which may require some time to become available depending on your client.
+
+* Web redirections are now handled more reliably, reducing the likelihood of hitting dead or invalid pages in the UI. Please ensure you force-refresh or clear any browser cache for your Jellyfin instance to see the full benefits.
+
+* DLNA support is now provided by a plugin and has been removed from the core server. This has several major benefits: first, it can be updated independently of the server to some degree; second, it will not be enabled for anyone unless they want it and explicitly install the plugin; and third, it reduces the potential of security holes around DLNA due to both previous benefits. If you wish to use (or continue to use) DLNA, please install the plugin after upgrading.
+
+* We now support AVIF and WEBP images for Pictures libraries.
+
+* Tags are now accounted for during searches, allowing one to search by tag.
+
+* We now support multiple simultaneous subtitle tracks (maximum of two, a primary and secondary) in the web player.
+
+* We've revamped the administrative dashboard UI to help improve usability and ease of finding options.
+
+### API & Security
+
+* All API endpoints now return proper return codes, ensuring that API endpoint results can be reliably interpreted without additional parsing.
+
+* Our API versioning scheme is now based off of the "last unstable build before a release" and uses a date-based version going forward, to help make it a bit clearer and assist in automating the circular dependencies in our API specifications.
+
+* Parental ratings are significantly improved, with better enforcement, inheritable ratings, and more.
+
+* LiveTV and Collection permissions are now discrete and configurable per-user.
+
+* All playback endpoints now enforce authentication, preventing anonymous users from streaming by potentially guessing item IDs (highly unlikely, but not impossible).
```

Comment:
I don't believe this is true, eg `/Vidoes/{itemId}/stream` doesn't require auth

Reply:
Correct, this is not in 10.9. Relevant PR: https://github.com/jellyfin/jellyfin/pull/9873

Reply:
OK it was stated in the list, I guess by @Shadowghost assuming it would be merged by now :) I've just removed this line.